internal/format/format.go

```diff
@@ -126,13 +126,12 @@ func generatePackage(file *os.File, pkg models.Package) {
 }
 
 func buildDocument(module models.Module) (*models.Document, error) {
-	uuid := uuid.New().String()
 	return &models.Document{
 		SPDXVersion:       "SPDX-2.2",
 		DataLicense:       "CC0-1.0",
 		SPDXID:            "SPDXRef-DOCUMENT",
 		DocumentName:      module.Name,
-		DocumentNamespace: fmt.Sprintf("http://spdx.org/spdxpackages/%s-%s-%s", module.Name, module.Version, uuid),
+		DocumentNamespace: buildNamespace(module.Name, module.Version),
```

Comment:
is it any reason to miss do the same join Name+Version for DocumentName? 
DocumentName is expected to have a version based on the spec

Reply:
it's not required on the SPDX spec https://spdx.github.io/spdx-spec/2-document-creation-information/#24-document-name, this is our internal format, but if a package does not have a version it will show double dash on the namespace, and QA reported it as bug